datastore/dynamo.go

```diff
@@ -0,0 +1,57 @@
+package datastore
+
+import (
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+)
+
+const (
+	// Strings for the primary key
+	pk     string = "ClientID"
+	projPk string = "ClientID, ID"
+
+	// Strings for (ClientID, DataTypeMtime) GSI
+	clientIDDataTypeMtimeIdx   string = "ClientIDDataTypeMtimeIndex"
+	clientIDDataTypeMtimeIdxPk string = "ClientID"
+	clientIDDataTypeMtimeIdxSk string = "DataTypeMtime"
+
+	// Strings for (ID, ExpireAt) GSI
+	idExpireAtIdx   string = "IDExpireAtIndex"
+	idExpireAtIdxPk string = "ID"
+	idExpireAtIdxSk string = "ExpireAt"
+)
+
+var (
+	// Table is the name of the table in dynamoDB, could be modified in tests.
+	Table string = os.Getenv("TABLE_NAME")
```

Comment:
Make this a configuration, and pass it into the datastore structure on NewDynamo?

Reply:
Follow up created at #2.